pkg/reconciler/autoscaling/kpa/controller.go

```diff
@@ -37,6 +37,7 @@ import (
 	"knative.dev/pkg/logging"
 	pkgreconciler "knative.dev/pkg/reconciler"
 	"knative.dev/serving/pkg/apis/autoscaling"
+	av1alpha1 "knative.dev/serving/pkg/apis/autoscaling/v1alpha1"
```

Comment:
This was likely the bad boy in the older PR. This was importing networking there.